README.md

```diff
@@ -85,6 +85,29 @@ In `config/initializers/devise.rb`:
     # end
     config.saml_update_user = true
 
+    # Lambda that is called if Devise.saml_update_user and/or Devise.saml_create_user are true.
+    # Receives the model object, saml_response and auth_value, and defines how the object's values are
+    # updated with regards to the SAML response. 
+    # config.saml_update_resource_hook = -> (model, saml_response, auth_value) {
+    #   saml_response.attributes.resource_keys.each do |key|
+    #     user.send "#{key}=", saml_response.attribute_value_by_resource_key(key)
+    #   end
+    #
+    #   if (Devise.saml_use_subject)
+    #     model.send "#{Devise.saml_default_user_key}=", auth_value
+    #   end
+    #
+    #   model.save!
+    # }
```

Comment:
There's a hanging `user` in here, but I actually think this might be better if we keep `user` for the object name—that distinguishes it from the `model` *class* in the below example. It's also consistent with the other option names that mention "user."

```suggestion
    # config.saml_update_resource_hook = -> (user, saml_response, auth_value) {
    #   saml_response.attributes.resource_keys.each do |key|
    #     user.send "#{key}=", saml_response.attribute_value_by_resource_key(key)
    #   end
    #
    #   if (Devise.saml_use_subject)
    #     user.send "#{Devise.saml_default_user_key}=", auth_value
    #   end
    #
    #   user.save!
    # }
```